object-manipulation-2/src/invert.js

```diff
@@ -0,0 +1,18 @@
+/* exported invert */
+
+// get value Object.values()
+// create variable vor values
+// create variable for counter to iterate over array
+// create for in loop to iterate over object
+// each iteration assing new key/value pair
+
+function invert(source) {
+  var newArray = Object.values(source);
+  var newObject = {};
+  var counter = 0;
+  for (var property in source) {
+    newObject[newArray[counter]] = property;
+    counter++;
```

Comment:
This code can be simplified a bit to leave out `Object.values()`.

```js
newObject[source[property]] = property;
```

Reply:
yea, this newArray variable was completely unnecessary. I fixed it